src/main/java/com/minecolonies/coremod/colony/managers/ColonyPackageManager.java

```diff
@@ -92,59 +89,28 @@ public void setLastContactInHours(final int lastContactInHours)
     @Override
     public Set<EntityPlayerMP> getSubscribers()
```

Comment:
Rename the method to what it returns in the new scenario: getSubscribersByCloseness